housingbayarea/bloom
# feat: release lottery apps fe

This PR releases the following commits 

- feat: application side nav https://github.com/bloom-housing/bloom/pull/4237
- feat: feature flag and new endpoint https://github.com/bloom-housing/bloom/pull/4252
- fix: build resolution https://github.com/bloom-housing/bloom/pull/4278
- fix: clean up dto https://github.com/bloom-housing/bloom/pull/4281
- feat: lottery public list view https://github.com/bloom-housing/bloom/pull/4286

## How Can This Be Tested/Reviewed?

This PR can be tested by creating a public user and applying to listings that are open (with and without application due dates), closed (with and without lottery dates), and lottery run (triggered by publishing to public. Ensure the side nav, statues and buttons all show as expected.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
